firefox_puppeteer/ui/toolbars.py

```diff
@@ -114,7 +114,7 @@ def focus(self, evt='click'):
         """Focus the location bar according to the provided event.
 
         :param evt: The event to synthesize in order to focus the urlbar
-                    (one of 'click' or 'shortcut').
+                    (one of ``click`` or ``shortcut``).
```

Comment:
Restructured Text simply italicizes items enclosed with a single tick, while Markdown treats them as inline code. But I'm not entirely sure what's right here, so I'll defer to your suggestion.